server/server.js

```diff
@@ -1,37 +1,36 @@
 const express = require('express');
-const cors = require('cors');
+//const cors = require('cors');
```

Comment:
do we need this? and line 10

Reply:
We do need this for development. You cannot get data from backend without it.